app/assets/stylesheets/refinery/home.scss

```diff
@@ -213,20 +197,199 @@
         letter-spacing: 0.0035rem;
         margin-top: 1.4rem;
         padding: 1rem 3.1rem;
+        text-align: center;
         text-decoration: none;
         text-transform: uppercase;
+        transition: all 0.4s ease-in-out;
         white-space: nowrap;
         width: 12.625rem;
-
+        
         a {
           text-decoration: none;
         }
 
+        &:hover {
+          background-color: $v3-color_green-light;
+          border-color: $v3_color_bg-lightest;
+          cursor: pointer;
+          
+          a {
+            color: $v3_color_bg-lightest;
+          }
+        }
+
         @include media(medium) {
-          width: 9.9375rem;
           padding: 0.59rem 1.75rem;
+          width: 9.9375rem;
+        }
+      }
+    }
+
+    &--links {
+      grid-column: 3;
+      grid-row: 1;
+      margin: 5.2rem 0 0;
+
+      ul > li {
+        list-style: none;
+        padding-bottom: 0.25em;
```

Comment:
Your mixing `em`s and `rem`s here...let's put this on the Friday lunch topic. :)